### PR TITLE
feat(angular/notification): add type `info-light`

### LINKED ISSUE #1560: Feature request: Info Notification vom Lean Design in Standard Design aufnehmen
When developing the webshop-offers UI with the angular component library, I stumbled over a 'missing' component, that was described in the UX Design.
The component in question is on the right side of this screenshot:
image

This component is only available in lean design, that we cannot (and do not want to) use for our public UI, but our UIX Specialist wants to have this design in the standard collection.

Please provide the lean styled info component also in the standard design.

### PR DESCRIPTION
**Pull request by mhaertwig**
Closes #1560

### CODE REVIEW

**Review by kyubisation — APPROVED** [review]
LGTM 👍

File: src/angular/notification/notification.ts [hunk 1]
```diff
@@ -126,6 +127,8 @@ export class SbbNotification extends _SbbNotificationMixinBase implements OnChan
           return variant === 'standard' ? 'kom:sign-exclamation-point-small' : 'kom:sign-x-small';
         } else if (this.type === 'warn') {
           return 'kom:sign-exclamation-point-small';
+        } else if (this.type === 'info-light') {
```
Comment (jeripeierSBB) [inline src/angular/notification/notification.ts:130]: Can be deleted as it is standard icon

File: src/components-examples/angular/notification/simple-notification/simple-notification-example.html [hunk 2]
```diff
@@ -2,3 +2,6 @@
 <sbb-notification type="info">Info</sbb-notification>
 <sbb-notification type="warn">Warn</sbb-notification>
 <sbb-notification type="error">Error</sbb-notification>
+<sbb-notification type="info-light"
```
Comment (jeripeierSBB) [inline src/components-examples/angular/notification/simple-notification/simple-notification-example.html:5]: What about placing it after info? If yes, please also update code occurrences

**Review by jeripeierSBB — APPROVED** [review]
LGTM thanks

**Pull request merged**